images/utils.go

```diff
@@ -24,19 +21,14 @@ type forEachImageFunc func(index int, image Image) error
 //
 //nolint:revive
 func getImage(ctx context.Context, l logging.Logger, imageProvider Provider, alert types.Alert) (*Image, error) {
-	token := getTokenFromAnnotations(alert.Annotations)
-	if token == "" {
-		return nil, nil
-	}
-
 	ctx, cancelFunc := context.WithTimeout(ctx, ProviderTimeout)
 	defer cancelFunc()
 
-	img, err := imageProvider.GetImage(ctx, token)
+	img, err := imageProvider.GetImage(ctx, alert)
 	if errors.Is(err, ErrImageNotFound) || errors.Is(err, ErrImagesUnavailable) {
 		return nil, nil
 	} else if err != nil {
-		l.Warn("failed to get image with token", "token", token, "error", err)
+		l.Warn("failed to get image", "error", err)
```

Comment:
Suggestion for extra context:
```suggestion
		l.Warn("failed to get image", "fingerprint", alert.Fingerprint(), "error", err)
```
Either that or `alert.String()`.

Reply:
`alert.String()` is already in the context of this logger from `WithStoredImages`